pkg/sql/ir_generator.go

```diff
@@ -124,7 +136,38 @@ func generateTrainStmtByModel(slct *parser.SQLFlowSelectStmt, connStr, cwd, mode
 	if err != nil {
 		return nil, err
 	}
-	return generateTrainStmtWithInferredColumns(slctWithTrain, connStr)
+	return generateTrainStmtWithInferredColumns(slctWithTrain, connStr, false)
```

Comment:
Why we are not verifying the label field of the trainSQL using prediction?

Reply:
Predict SQL may not select the label column in the SELECT statement, we need only verify the feature columns only in that case.